src/grove/management/commands/load_predicates.py

```diff
@@ -0,0 +1,43 @@
+from csv import reader
+from logging import getLogger
+
+from django.core.management.base import BaseCommand, CommandError
+from plastron.namespaces import namespace_manager
+from rdflib.util import from_n3
+from vocabs.models import Predicate
+
+logger = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = """
+           A small management utility that reads a simple, 2-column CSV file (predicate, object_type),
+           and does a get_or_create for each predicate listed.
+           """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-f",
+            "--file",
+            help="The 2-column CSV file (predicate, object_type)",
+            action="store",
+            type=str,
+            required=True,
+        )
+
+    def handle(self, *args, **options):
+        try:
+            with open(options["file"], "r", newline="") as csv_file:
+                csv_reader = reader(csv_file)
```

Comment:
I would suggest using the [csv.DictReader](https://docs.python.org/3.11/library/csv.html#csv.DictReader) class, and explicitly specifying the column names:

```python
from csv import DictReader

csv_reader = DictReader(csv_file, fieldnames=('predicate', 'object_type'))
try:
    for row in csv_reader:
        logger.debug(str(row))
        uri = from_n3(row['predicate'], nsm=namespace_manager)
        Predicate.objects.get_or_create(uri=uri, object_type=row['object_type'])
except ...:
    ...
```